src/app/patient-summary/components/PatientSummary.tsx

```diff
@@ -0,0 +1,46 @@
+"use client";
+import * as React from "react";
+import Tabs from "@mui/material/Tabs";
+import Tab from "@mui/material/Tab";
+import Box from "@mui/material/Box";
+import fhirBundleJson from "../sample/bundle.json"
+import TabPanel from "./TabPanel";
+import { useState } from "react";
+
+const dataTabs = Array.from(
+  new Set(fhirBundleJson.entry.map((entry) => entry.resource.resourceType))
+);
+
+const renderTabs = () =>
+  dataTabs.map((resourceType) => (
+    <Tab label={resourceType} key={resourceType} value={resourceType} />
+  ));
```

Comment:
Wondering if we should move this function into the main component function as was done with the renderPanels?

Reply:
Yes, it makes sense.